lib/sassc/script.rb

```diff
@@ -30,10 +30,18 @@ module Script
 require 'sass/util'
 require 'sass/script/value/base'
 require 'sass/script/value/string'
+require 'sass/script/value/number'
 require 'sass/script/value/color'
+require 'sass/script/value/map'
 
 SassC::Script::String = Sass::Script::Value::String
 SassC::Script::Value::String = Sass::Script::Value::String
 
+SassC::Script::Number = Sass::Script::Value::Number
+SassC::Script::Value::Number = Sass::Script::Value::Number
+
 SassC::Script::Color = Sass::Script::Value::Color
 SassC::Script::Value::Color = Sass::Script::Value::Color
+
+SassC::Script::Map = Sass::Script::Value::Map
+SassC::Script::Value::Map = Sass::Script::Value::Map
```

Comment:
I need to make this more clear in the code, but we do not need to include `SassC::Script::Number` or `SassC::Script::Map` here.   

I have `SassC::Script::String`  and `SassC::Script::Color` here because of a mistake that I need to be compatible with.